alacritty/Cargo.toml

```diff
@@ -12,15 +12,13 @@ rust-version = "1.74.0"
 
 [dependencies.alacritty_terminal]
 path = "../alacritty_terminal"
-version = "0.24.2-dev"
+version = "0.24.2-rc1"
 
 [dependencies.alacritty_config_derive]
-path = "../alacritty_config_derive"
-version = "0.2.5-dev"
+version = "0.2.4"
 
 [dependencies.alacritty_config]
-path = "../alacritty_config"
-version = "0.2.3-dev"
+version = "0.2.2"
```

Comment:
I'm not sure what to do with this btw, one option would be to manually revert the version bump on this branch and include the way we did before?

The issue is that they are in the lock file, but I pull them from crates.io, since we don't actually use `-dev` versions.

Reply:
I've rolled back the versions to be stable and have a path, so we don't have `-dev` crates in a release.